src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -656,28 +656,42 @@ private async Task<bool> EnumerateFilesFromArtifactsProvider(TContext globalCont
 
                 string filePath = artifact.Uri.GetFilePath();
 
+                long artifactSizeInBytes = -1;
+                try
+                {
+                    artifactSizeInBytes = artifact.SizeInBytes.Value;
+                }
+                catch (Exception e) when (e is ArgumentException || (e is IOException && e.Message.IndexOf("The filename, directory name, or volume label syntax is incorrect.") != -1))
```

Comment:
>Exception [](http://example.com/codeflow?start=23&length=9)

this line of code is very long, perhaps create a string const for your message literal.